tests/orbit/models/test_ktrlite.py

```diff
@@ -0,0 +1,162 @@
+import pytest
+import numpy as np
+import pandas as pd
+
+from orbit.estimators.stan_estimator import StanEstimatorMAP
+from orbit.models.ktrlite import BaseKTRLite, KTRLiteMAP
+from orbit.diagnostics.metrics import smape
+
+
+@pytest.mark.parametrize("estimator_type", [StanEstimatorMAP])
```

Comment:
since this is the only parameter, do we still need this line?

Reply:
let's keep as it is for now, since it doesn't hurt. just in case in the future we may incorporate new estimator type and test it out.